content/posts/newsletter/2019-10-01.md

```diff
@@ -0,0 +1,124 @@
+---
+title: The XMPP Newsletter, 01 Oct 2019: FOSDEM 2020, modernization of XMPP, peer networks
+save_as: 2019/10/newsletter-01-october/index.html
+url: 2019/10/newsletter-01-october/
+date: 2019-10-01 09:00
+author: nyco
+blog_id: newsletter
+category: Newsletter
+---
+
+Welcome to the XMPP newsletter October edition, covering the month of September.
+
+New this month: we added the license (CC by-sa 4.0), and the credits as this is a community effort.
```

Comment:
> we added the license

That feels like an incomplete sentence. Maybe: "we've made explicit that this newsletter can be shared and adapted as defined by the CC by-sa license"  (and include a link to https://creativecommons.org/licenses/by-sa/4.0/ )

Reply:
Thx, fixed